docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,65 @@
+# Release 465 (14 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in [EWKB
+  format](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry)
+  with the {func}`ST_GeomFromBinary` function. ({issue}`23824`)
+* Add support for the `ORDER BY` clause in a windowed aggregate function. ({issue}`23929`)
+* Improve performance of queries with selective joins on large build sides. ({issue}`22824`)
+* Fix potential configuration leakage between catalogs using JDBC-based
+  connectors. ({issue}`24058`)
+* Lower probability of queries failing when running with task retries due to
+  running out task descriptors memory. ({issue}`24114`)
+
+## CLI
+
+* Fix quoting of output values when `CSV_UNQUOTED` or `CSV_HEADER_UNQUOTED`
```

Comment:
What's wrong with the quoting? If we're still including quotes despite it needing to be unquoted, maybe "Fix incorrect quotation of output values"